### ISSUE TITLE
Bug, maybe when saving shapes layer with no shapes?

### ISSUE THREAD

**Issue by tlambert03** [napari/napari-svg#10]
just got this error when saving a whole bunch of layers.  I'm not certain, but I think it might have been due to shapes layer with nothing in it?

```pytb
In [3]: viewer.layers.save('test', selected=False)                                                                                                                       
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
napari/plugins/io.py in _write_multiple_layers_with_plugins(path, layers, plugin_name, plugin_manager)
    216         writer_function = result.result
--> 217         implementation = result.implementation
    218 

site-packages/napari_svg/hook_implementations.py in writer(path, layer_data)
     85         function_string = ld[2] + '_to_xml(ld[0], ld[1])'
---> 86         xml_list, extrema = eval(function_string)
     87         full_xml_list += xml_list

site-packages/napari_svg/hook_implementations.py in <module>

site-packages/napari_svg/layer_to_xml.py in shapes_to_xml(data, meta)
    262     # Find extrema of data
--> 263     mins = np.min([np.min(d, axis=0) for d in shapes], axis=0)
    264     maxs = np.max([np.max(d, axis=0) for d in shapes], axis=0)

<__array_function__ internals> in amin(*args, **kwargs)

site-packages/numpy/core/fromnumeric.py in amin(a, axis, out, keepdims, initial, where)
   2791     """
-> 2792     return _wrapreduction(a, np.minimum, 'min', axis, None, out,
   2793                           keepdims=keepdims, initial=initial, where=where)

site-packages/numpy/core/fromnumeric.py in _wrapreduction(obj, ufunc, method, axis, dtype, out, **kwargs)
     89 
---> 90     return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
     91 

ValueError: zero-size array to reduction operation minimum which has no identity

The above exception was the direct cause of the following exception:

PluginCallError                           Traceback (most recent call last)
<ipython-input-3-5755ed3829a0> in <module>
----> 1 viewer.layers.save('test', selected=False)

napari/components/layerlist.py in save(self, path, selected, plugin)
    256             return []
    257 
--> 258         return save_layers(path, layers, plugin=plugin)

napari/plugins/io.py in save_layers(path, layers, plugin)
    133         File paths of any files that were written.
    134     """
--> 135     if len(layers) > 1:
    136         written = _write_multiple_layers_with_plugins(
    137             path, layers, plugin_name=plugin

napari/plugins/io.py in _write_multiple_layers_with_plugins(path, layers, plugin_name, plugin_manager)
    217         implementation = result.implementation
    218 
--> 219     if not callable(writer_function):
    220         if plugin_name:
    221             msg = f'Requested plugin "{plugin_name}" is not capable'

PluginCallError: Error in plugin 'svg', hook 'napari_get_writer': zero-size array to reduction operation minimum which has no identity
```

**Comment by sofroniewn** [comment]
ah ok, doesn't surprise me something weird may happen there, thanks for flagging. i need to look into it more

**Comment by sofroniewn** [comment]
Actually @kevinyamauchi can you look into this as you prepare the `0.1.3` release of `napari-svg`

**Comment by kevinyamauchi** [comment]
No problem!

**Comment by kevinyamauchi** [comment]
As @tlambert03 suspected, the issue is due to trying to write out an empty shapes layer. What is the expected behavior here?

Do we have some way to tell the writer to skip the layer (e.g., something like a pass or continue).

Or should we write an SVG with an arbitrarily-sized blank canvas?

**Comment by sofroniewn** [comment]
Hmm, two things to consider are what to do if someone tries to save only an empty shapes layer. Here I'm not so sure, maybe writing an empty canvas or something is fine?

The other thing is if someone is trying to save a list of layers which includes an empty shapes layer. Here we definitely don't want to trip everything up, so adding an "empty" xml element seems good.

One option could be to have this function return extrema that are np.nan
https://github.com/napari/napari-svg/blob/master/napari_svg/layer_to_xml.py#L263-L264

Then we need make this min/ max handle np.nan https://github.com/napari/napari-svg/blob/master/napari_svg/hook_implementations.py#L90-L93

and then make this function do something with np.nan https://github.com/napari/napari-svg/blob/master/napari_svg/xml_to_svg.py#L21-L22

How does this sound? I'm very open here to other ideas

**Comment by kevinyamauchi** [comment]
I implemented @sofroniewn's suggestion in #12
